src/ContactForm/ContactForm.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+
+
+// import styling 
+import './ContactForm.scss';
+
```

Comment:
Typically components are named with a uppercase but sass partials normally start with a _ and should be @import 'ed into a main styles.scss to avoid specific importing on different components.

Reply:
Fair approach ... I was taught that everything related to component stays with the component to simplify maintenance. I like that approach but I can see where I could lose things like mixins and bigger styling opportunities.